Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    decanter (4.0.0)
+    decanter (4.0.1)
```

Comment:
I believe this is a `minor` change (an enhancement) rather than a bug fix (patch)